bing.com.BingWebMasterDPOO.json

```diff
@@ -0,0 +1,22 @@
+{
+  "providerId": "bing.com",
+  "providerName": "Microsoft Bing",
+  "serviceId": "BingWebMasterDPOO",
+  "serviceName": "Domain proof of ownership with Bing Webmaster services",
+  "version": 1.2,
+  "logoUrl": "https://domainconnect.org/wp-content/uploads/<<bing logo>> link",
```

Comment:
Mind adding a valid URL.

Reply:
You can also omit it.  The field isn't required.